shiny/render/transformer/_transformer.py

```diff
@@ -294,6 +300,24 @@ async def _run(self) -> OT:
         )
         return ret
 
+    def _repr_htmltools_(self) -> str | None:
```

Comment:
Is this function used anywhere?

Reply:
Yeah, my JupyterLab extension registers it as a new format.

Reply:
Never mind, removed this and handle it a different way now